src/test/java/com/ait/lienzo/client/core/shape/wires/handlers/impl/WiresConnectorHandlerImplTest.java

```diff
@@ -132,15 +156,14 @@ public void testOnNodeMouseClick() {
         tested.onNodeMouseClick(event);
         verify(clickTimer, times(1)).cancel();
         verify(clickTimer, times(1)).schedule(anyInt());
-        verify(doubleClickEventConsumer, never()).accept(any(Event.class));
+        verify(mouseDownConsumer, never()).accept(any(Event.class));
     }
 
     @Test
-    public void testOnNodeMouseDoubleClick() {
-        NodeMouseDoubleClickEvent event = mock(NodeMouseDoubleClickEvent.class);
-        tested.onNodeMouseDoubleClick(event);
-        verify(doubleClickEventConsumer, times(1)).accept(any(Event.class));
-        verify(clickEventConsumer, never()).accept(any(Event.class));
+    public void testAddControlPoint() {
+        //TODO:improve the tests here
```

Comment:
Do you plan to do it within this PR or not?

Reply:
yes, I'll implement it, just waiting to get the lienzo-core, to avoid rework.